app/src/main/java/com/example/cocktaildb/screen/home/HomeContract.kt

```diff
@@ -0,0 +1,17 @@
+package com.example.cocktaildb.screen.home
+
+import com.example.cocktaildb.data.model.Cocktail
+import com.example.cocktaildb.utils.base.BasePresenter
+
+interface HomeContract {
+    interface View {
+        fun showCocktails(cocktails: List<Cocktail>)
+        fun showError(message: String)
+        fun showLoading()
+        fun hideLoading()
+    }
+    
+    interface Presenter : BasePresenter<View> {
+        fun loadCocktails()
+    }
+} 
```

Comment:
```suggestion
} 

```
check tương tự vs những file khác